packages/cf-component-form/src/FormLabel.js

```diff
@@ -1,24 +1,23 @@
 import React, { PropTypes } from 'react';
+import { createComponent } from 'cf-style-container';
 
-class FormLabel extends React.Component {
-  render() {
-    let className = 'cf-form__label';
+const styles = ({ theme, hidden }) => ({
+  fontSize: theme.fontSize,
+  display: hidden ? 'none' : 'block',
+  marginBottom: theme.marginBottom,
+  color: theme.color
+});
 
-    if (this.props.hidden) {
-      className += ' cf-form__label--hidden';
-    }
-
-    return (
-      <label className={className}>
-        {this.props.children}
-      </label>
-    );
-  }
-}
+const FormLabel = ({ children, className }) => (
```

Comment:
Well, if that's everyone's opinion you should open a new PR and revert it everywhere since we use functional components within all new cf-ui codebase. I don't like it. If we really need refs, we can do it on per instance basis.